<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order 통계</string>
  </property>
  <widget class="QPushButton" name="btn_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>엑셀 파일 열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>28</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;파일명: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>991</width>
     <height>421</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <row>
    <property name="text">
     <string>필터</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>요일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>오전/오후</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>요청 경로</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>섭외내용</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>섭외종류</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>업체성격</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>업체명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이메일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>연락처</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>진행여부</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>진행완료</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_fileName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>27</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_init">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>필터 초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_monthly_report">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>월별 총 집계</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total_count">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>104</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;총 데이터 수 : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_cols">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>61</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_monthly_item_report">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>항목별 집계</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
